axelor-helpdesk/src/main/java/com/axelor/apps/helpdesk/db/repo/TicketManagementRepository.java

```diff
@@ -22,12 +22,22 @@
 import com.axelor.apps.base.service.app.AppBaseService;
 import com.axelor.apps.base.service.exception.TraceBackService;
 import com.axelor.apps.helpdesk.db.Ticket;
+import com.axelor.apps.helpdesk.db.TicketStatus;
 import com.axelor.apps.helpdesk.service.TicketService;
+import com.axelor.apps.helpdesk.service.TicketStatusService;
+import com.axelor.apps.helpdesk.service.app.AppHelpdeskService;
+import com.axelor.studio.db.AppHelpdesk;
 import com.google.inject.Inject;
+import java.util.Map;
+import java.util.Optional;
 
 public class TicketManagementRepository extends TicketRepository {
 
   @Inject private TicketService ticketService;
+
+  @Inject private TicketStatusService ticketStatusRepository;
+
+  @Inject private AppHelpdeskService appHelpdeskService;
   @Inject private AppBaseService appBaseService;
```

Comment:
Since this is a feature, we might as well remove the injections, and use as usual, the Beans.get() to call services.